src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdUpdateService.java

```diff
@@ -357,6 +360,18 @@ public void incrementOrgAssignedUsersSupplementaryData(final String caseId,
         submitSupplementaryDataUpdateToCcd(caseId, authorisation, serviceAuth, supplementaryData);
     }
 
+    public void submitSupplementaryDataToCcdForServiceID(final String caseId,
+                                                           final String authorisation,
+                                                           final String serviceAuth) {
+        log.info("submitSupplementaryDataToCcdForServiceID Case ID: {}", caseId);
+        Map<String, Map<String, Map<String, Object>>> supplementaryData = new HashMap<>();
+        supplementaryData.put("supplementary_data_updates",
+            singletonMap(SET_OPERATION,
+                singletonMap(SERVICE_ID_FIELD, SERVICE_ID)));
```

Comment:
This looks like it should work to avoid overwriting org information in the supplementary data. It might be worth us having a test case for Elis to confirm, as it could be a bad bug and time-consuming to fix if we did overwrite any org info by mistake